tss-esapi/tests/integration_tests/abstraction_tests/ek_tests.rs

```diff
@@ -18,12 +16,16 @@ fn test_retrieve_ek_pubcert() {
     // did not exist (the test is run on a TPM without an endorsement cert), it still passes.
     match ek::retrieve_ek_pubcert(&mut context, AsymmetricAlgorithm::Rsa) {
         Ok(_) => (),
-        Err(Error::Tss2Error(Tss2ResponseCode::FormatOne(FormatOneResponseCode(395)))) => (),
+        Err(Error::TssError(ReturnCode::Tpm(TpmResponseCode::FormatOne(error)))) => {
+            assert_eq!(error.error_number(), TpmFormatOneError::Handle)
+        }
```

Comment:
```rust
        Err(Error::TssError(ReturnCode::Tpm(TpmResponseCode::FormatOne(error)))) if error.error_number() == TpmFormatOneError::Handle => ()
```

Reply:
Yeah, that would work to. Not sure if I find it better though.